antora-ui-camel/src/css/nav.css

```diff
@@ -118,6 +118,32 @@ html.is-clipped--nav {
   overflow-y: auto;
 }
 
+.nav-menu::-webkit-scrollbar {
+  width: 0.42rem;
+}
+
+.nav-menu::-webkit-scrollbar-track {
+  background: var(--nav-scrollbar-color);
+  border-radius: 3.125rem;
+}
+
+.nav-menu::-webkit-scrollbar-thumb {
+  background: var(--nav-scroll-thumb-color);
+  border-radius: 3.125rem;
+}
+
+.nav-menu::-webkit-scrollbar-thumb:hover,
+.nav-menu::-webkit-scrollbar-thumb:active {
+  background: var(--nav-scrollbar-active-color);
+}
+
+@media screen and (max-width: 1023px) {
+  .nav-menu::-webkit-scrollbar {
+    width: 0;
+    background: transparent;
+  }
+}
+
```

Comment:
If we could make this changes align with [the Default Antora UI](https://gitlab.com/antora/antora-ui-default/-/blob/master/src/css/nav.css#L91-105), that would be awesome, much easier to rebase then.

Reply:
The thing is that in the default antora UI, they make use of the `nav-panel-menu` however in our case, we make use of the `nav`  'cause under the component section, we need to avoid the scrollbar for the `quick lookup` as it is included within `nav-panel-menu`.